CHANGELOG.md

```diff
@@ -1,3 +1,130 @@
+ * Audit permissions on ACS UI ([#35661](https://projects.theforeman.org/issues/35661), [5f4ffd28](https://github.com/Katello/katello.git/commit/5f4ffd28caf97e43a2471cbd97ebd6baa22422fb))
+ * Change 'Subscription Allocation' to 'Manifest' on the Manage Manifest screen ([#35618](https://projects.theforeman.org/issues/35618), [723f67e8](https://github.com/Katello/katello.git/commit/723f67e87aa8d482be64a8e7dda054f93f98ae96))
+ * ACS UI - General updates ([#35571](https://projects.theforeman.org/issues/35571), [3dd05eaa](https://github.com/Katello/katello.git/commit/3dd05eaae500ace69b76ea590ce3d882ed1510c8))
+ * ACS Wizard - UX changes ([#35565](https://projects.theforeman.org/issues/35565), [e2314970](https://github.com/Katello/katello.git/commit/e231497012e63bb2589d9cc19c146f21656cc3dc))
+ * When searching for content, dropdown filters are literal search terms. ([#35512](https://projects.theforeman.org/issues/35512), [76f715bf](https://github.com/Katello/katello.git/commit/76f715bfd28a50c59e94f924a7d03be91ef512b2))
+
+### Inter Server Sync
+ * Generated content views are displayed in Main Dashboard ([#35723](https://projects.theforeman.org/issues/35723), [6b5e6ee5](https://github.com/Katello/katello.git/commit/6b5e6ee53f1453407b3ec47874e11e9b440f9b63))
+ * Content View Versions generated by Export are still listed in Composite page ([#35501](https://projects.theforeman.org/issues/35501), [3b23b993](https://github.com/Katello/katello.git/commit/3b23b99334ef9a3bd8d7d298b84c51dedf9806a2))
+ * Syncable exports not properly validated ([#35442](https://projects.theforeman.org/issues/35442), [19056c10](https://github.com/Katello/hammer-cli-katello.git/commit/19056c10c202374a8b4cf200f357b299555e8ce6), [2166e5e3](https://github.com/Katello/katello.git/commit/2166e5e34c2013f419036c9f24f1dfcffef8e475))
+ * Importing a custom repository with different label but same name causes validation error ([#35425](https://projects.theforeman.org/issues/35425), [cf159cad](https://github.com/Katello/katello.git/commit/cf159cadb0b306faea429aba0af5fa07ad45b105))
+ * Pathing issue on exports ([#35410](https://projects.theforeman.org/issues/35410), [491e8af0](https://github.com/Katello/katello.git/commit/491e8af044cae8da1cb7b2c77f33516dda79aab3))
+ * Need to be able to provide custom cert for ISS for Red Hat CDN ([#35296](https://projects.theforeman.org/issues/35296), [a0e63cb0](https://github.com/Katello/katello.git/commit/a0e63cb063a07646ef079acfab762a6e7435b110), [6a47ac35](https://github.com/Katello/hammer-cli-katello.git/commit/6a47ac3546be9aafa2533e09b9ca4b51fd449f43))
+
+### Container
+ * Getting "undefined method `schema_version' for nil:NilClass" while syncing from quay.io ([#35709](https://projects.theforeman.org/issues/35709), [475388ea](https://github.com/Katello/katello.git/commit/475388eaef0796339326b17b8362b5015c719f9c))
```

Comment:
Ruby error messages break Markdown.. :confounded:

Reply:
It looks fine in Chrome, are you seeing it not render at all?

Reply:
I will push up a rebased commit, I removed the `'` `` and `nil:`